setup.py

```diff
@@ -15,10 +15,12 @@
     "numpy>=1.20.0,<2;python_version<'3.10'",
     "numpy>=1.23.3,<2;python_version>='3.10'",
     "pandas>=1.1.3;python_version<'3.10'",
-    "pandas>=1.3.4;python_version>='3.10'",
+    "pandas>=1.3.4;python_version=='3.10'",
+    "pandas>=1.5.0;python_version>'3.10'",
     "scikit-learn>=1.1.3,<2;python_version>='3.10'",
     "torch>=1.8.0;python_version<'3.10'",
-    "torch>=1.11.0;python_version>='3.10'",
+    "torch>=1.11.0;python_version=='3.10'",
```

Comment:
what happens if the python version is 3.10.x? does it fall under this line or the next one?

Reply:
I did some testing and I think `3.10.x == 3.10`.